### PR TITLE
Working branch (CSS navigation bar

### PR DESCRIPTION
**Pull request by 3Z3QU13L5**
@fxisco following your tips/comments I made some modification and also, added the css for the mobile navigation bar, it is a collapsing sideBar, since it doesn't have the javascript yet you only need to change `#navBar{ width: 50%;} -> { width: 0;} ` to removed

### CODE REVIEW

File: Index.html [hunk 1]
```diff
@@ -8,31 +8,33 @@
...
-            <li class="navBar__list">
-              <a href="#nowhere" title="Contact">Contact</a>
-            </li>
-          </ul>
-        </nav>
-      </div>
+      <nav>
+        <ul class="navBar__ul">
+          <li class="navBar__list__closeBtn">
+            <button class=" navBar__btn">&#9747</button>
```
Comment (fxisco) [inline Index.html]: remove white space from the beginning of the class

File: styleSheet.css [hunk 2]
```diff
@@ -5,12 +5,12 @@ ul {
...
   position: fixed;
   width: 100%;
-  height: 5%;
+  height: 5.5%;
   top: 0;
   background-color: #333;
 }
 
-#navBar>div{
+#navBar > nav{
```
Comment (fxisco) [inline styleSheet.css]: why do we need to wrap the nav inside a div called `#navBar`?

File: styleSheet.css [hunk 3]
```diff
@@ -29,22 +29,81 @@ ul {
...
-.navBar__list>a:hover{
+.navBar__list > a:hover{
   background-color: #111;
 }
 
-.navBar__list--active>a{
+.navBar__list--active > a{
   color: #4CAF50;
 }
+.navBar__list__closeBtn{
```
Comment (fxisco) [inline styleSheet.css]: Consistency in using `.navBar__list__closeBtn{` or `.navBar__list__closeBtn {`

File: styleSheet.css [hunk 4]
```diff
@@ -29,22 +29,81 @@ ul {
...
+.navBar__btn{
+  background-color: #111;
+  border: none;
+  color: #fff;
+  display: none;
+  font-size: calc(1em + 3.5vw);
+}
+
+.navBar__sidebarBtn{
+  height: 6.795%;
```
Comment (fxisco) [inline styleSheet.css]: this values look really odd? It is possible to use but is prefer to use integers

File: styleSheet.css [hunk 5]
```diff
@@ -29,22 +29,81 @@ ul {
...
+  position: fixed;
+  width: 12.08%;
+  left: 0;
+  top: 0;
+  opacity: 0.7;
+}
 
-#bodyResume{
+@media only screen and (max-width: 768px) {
+  .navBar__sidebarBtn,
```
Comment (fxisco) [inline styleSheet.css:79]: really cool grouping common styles 🎉

File: styleSheet.css [hunk 6]
```diff
@@ -29,22 +29,81 @@ ul {
...
+  height: 6.795%;
+  position: fixed;
+  width: 12.08%;
+  left: 0;
+  top: 0;
+  opacity: 0.7;
+}
 
-#bodyResume{
+@media only screen and (max-width: 768px) {
```
Comment (fxisco) [inline styleSheet.css:78]: did you pick up a approach to use in your design? `mobile-first` or `desktop-first`?

File: styleSheet.css [hunk 7]
```diff
@@ -29,22 +29,81 @@ ul {
...
+    transition: 0.5s;
+    width: 50%;
+    z-index: 1;
+  }
+  .navBar__list {
+    width: 100%;
+  }
+  .navBar__list > a{
+    display: block;
+    font-size: calc(1em + 2vw);
```
Comment (fxisco) [inline styleSheet.css:100]: are you having this fluid font size just for the nav? I think you can declare the font-size fluid for all the html

**Review by fxisco — COMMENTED** [review]
I see you declared `lang="en"` in the DOCTYPE tag, we should set it in the `<html>` tag.

File: styleSheet.css (hunk 3 above)
Comment (3Z3QU13L5) [reply styleSheet.css]: It took me a while to see the space

File: styleSheet.css (hunk 6 above)
Comment (3Z3QU13L5) [reply styleSheet.css:78]: Do you mean doing one complete version first? I have the complete idea of both, so I thought of working by section.

File: styleSheet.css (hunk 7 above)
Comment (3Z3QU13L5) [reply styleSheet.css:100]: it didn't cross my mind, will do.

File: styleSheet.css (hunk 2 above)
Comment (3Z3QU13L5) [reply styleSheet.css]: Well, I am using it so I can center the nav's text/links in the middle of the page

Comment (3Z3QU13L5) [reply styleSheet.css]: Yeah already fixed no more  nav inside a div

File: styleSheet.css (hunk 3 above)
Comment (fxisco) [reply styleSheet.css]: little details 👌

File: styleSheet.css (hunk 7 above)
Comment (fxisco) [reply styleSheet.css:100]: it is just an idea, we need to check how it works in small devices

File: styleSheet.css (hunk 6 above)
Comment (fxisco) [reply styleSheet.css:78]: I just want to make sure you know what approach you are using

In mobile first, our basic style is dedicated to mobile and we use media queries to upper sizes
e.g.

```css
body {
  background: red;
}

// This applies for >= 600px
@media (min-width: 600px) {
  body {
    background: green;
  }
}
```

But in a desktop fist approach our basic design is desktop and then we use media queries to targe smaller devices

```css
body {
  background: red;
}

//This applies for <= 720px
@media (max-width: 720px) {
  body {
    background: green;
  }
}

```

If you see the main difference is that with mobile first we always use `min-width` and with desktop first `max-width`

Comment (3Z3QU13L5) [reply styleSheet.css:78]: In that case, the approach I'm using would be `desktop-first`.

File: styleSheet.css (hunk 7 above)
Comment (3Z3QU13L5) [reply styleSheet.css:100]: I check I didn't found any weird behavior, but anyway I will be modifying it depending on the section or/and platform

**Pull request merged**